rl4co/envs/scheduling/ffsp/env.py

```diff
@@ -58,7 +57,7 @@ def __init__(
         generator_params: dict = {},
         **kwargs,
     ):
-        super().__init__(**kwargs)
+        super().__init__(check_solution=False, **kwargs)
```

Comment:
Is this always the case (no solution check)?

Reply:
uhm I think this is there just bc there is no check implemented for FFSP yet haha. Let me see if I can get one implemented

Reply:
No worries, this is not a pressing issue! Actually, it's even better to keep it to `False` during training for efficiency